.github/workflows/pull-request-meta.yml

```diff
@@ -19,12 +19,13 @@ jobs:
         submodules: false
 
     - name: Verify that postgres/ was not changed unintentionally
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
       shell: bash
       run: |
         required_prefix="Update bundled PostgreSQL"
 
-        title="${{ github.event.pull_request.title }}"
-        if [[ $title == $required_prefix* ]]; then
```

Comment:
Ohhh, this was not quoted so it was being interpreted as commands. Strange that it didn't happen before with my PR titles.

I shouldn't have done this in bash.

Reply:
The quoting around $title was my first suspicion as well, but that wasn't really it. It not being quoted means it will be expanded as *multiple* arguments to `[[`, which might be fine, though I quoted it to be safe.

The big issue was that backquotes are evaluated inside of double quotes, so when I had a PR title with backquotes in it, it got executed.
Fix was to pass it in as an env var.